windows-container-samples/msbuild/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM microsoft/windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+RUN (New-Object System.Net.WebClient).DownloadFile('http://download.microsoft.com/download/5/f/7/5f7acaeb-8363-451f-9425-68a90f98b238/visualcppbuildtools_full.exe', 'visualcppbuildtools_full.exe') ; \
+    Start-Process .\visualcppbuildtools_full.exe -ArgumentList '/NoRestart /S' -Wait ; \
+    rm visualcppbuildtools_full.exe
+
+COPY entrypoint.bat entrypoint.bat
+
+SHELL ["cmd"]
+
+WORKDIR /code
+
+ENTRYPOINT ["C:\\entrypoint.bat"]
```

Comment:
If the c++ toolchain really likes to be run in `cmd` and users are used to interacting with it using the "Visual Studio developer prompt", then I think sticking to `cmd` is good. But it also seems like MS is moving towards Powershell: http://news.softpedia.com/news/microsoft-replaces-command-prompt-with-powershell-in-latest-windows-10-build-510328.shtml